scripts/compile.js

```diff
@@ -23,7 +23,7 @@ async.waterfall([
 });
 
```

Comment:
I also see no reason to change the compile script in this PR, just revert the changes 😉

Reply:
I changed this one because it does not work under windows. Changing all the instances of `path` allows this to work in windows, but I can revert it if you prefer

Reply:
Yes, please revert it for now. I didn't know that it doesn't work with Windows, we should probably fix that. However, this would have to be a different pull request and I would suggest using `path.resolve` in that case (see [Path @ NodeJS docs](https://nodejs.org/docs/latest/api/path.html)).

Reply:
I agree. Reverted :)